WrightTools/collection/_collection.py

```diff
@@ -102,6 +102,25 @@ def create_collection(self, name="collection", position=None, **kwargs):
         setattr(self, name, collection)
         return collection
 
+    def convert(self, units, convert_variables=False, verbose=True):
+        """Convert units of a collection.
+        Parameters
+        ----------
+        units: string
+            Units to convert to.
+        """
+
+        for name in self.item_names:
+            item = self[name]
+            if isinstance(item, wt_data.Data):
+                item.convert(units, convert_variables=convert_variables, verbose=verbose)
+            elif isinstance(item, Collection):
+                Collection.convert(
+                    item, units, convert_variables=convert_variables, verbose=verbose
+                )
+            else:
+                pass
```

Comment:
@kameyer226 consider removing the isinstance checks. Collections and Data objects have the same function signature for `convert`, so you should be able to just blindly call without actually knowing what kind of item you have. I think so, at least :thinking: 

```suggestion
            item.convert(units, convert_variables=convert_variables, verbose=verbose)
```